controllers/hetznercluster_controller.go

```diff
@@ -260,6 +263,34 @@ func (r *HetznerClusterReconciler) reconcileNormal(ctx context.Context, clusterS
 	return reconcile.Result{}, nil
 }
 
+func setControlPlaneEndpoint(hetznerCluster *infrav1.HetznerCluster) bool {
```

Comment:
add a comment to this function
// setControlPlaneEndpoint updates hetznerCluster.Spec.ControlPlaneEndpoint and returns whether the hetznerCluster is ready or not